src/main/kotlin/com/mparticle/kits/MParticleAutopilot.kt

```diff
@@ -102,8 +100,24 @@ class MParticleAutopilot : Autopilot() {
             val accentColor = configuration.notificationColor
             if (!UAStringUtil.isEmpty(accentColor)) {
                 try {
-                    editor.putInt(NOTIFICATION_COLOR, Color.parseColor(accentColor))
+                    val colorValue = when {
+                        accentColor.equals("System default", ignoreCase = true) -> {
+                            val typedValue = TypedValue()
+                            val theme = context.theme
+                            theme.resolveAttribute(R.attr.windowBackground, typedValue, true)
+                            typedValue.data
+                        }
+
+                        else -> Color.parseColor(accentColor)
+                    }
+                    editor.putInt(NOTIFICATION_COLOR, colorValue)
                 } catch (e: IllegalArgumentException) {
+                    //  Handle parsing errors by setting the default color
+                    val typedValue = TypedValue()
+                    val theme = context.theme
+                    theme.resolveAttribute(R.attr.windowBackground, typedValue, true)
```

Comment:
Seeing these 3 lines duplicated with lines 105-107 is an opportunity for DRYing up.

Reply:
Since the issue is the same as mentioned above, removing this code.